### PR TITLE
EKS > Add tags in all resources

### PR DESCRIPTION
**Pull request by janavenkat**

### CODE REVIEW

File: eks/bastion.tf [hunk 1]
```diff
@@ -44,7 +48,8 @@ resource "aws_instance" "bastion" {
   monitoring = true
 
   tags = {
-    Name = "${var.environment_name}-bastion"
+    Name                  = "${var.environment_name}-bastion"
+    "KubespotEnvironment" = var.environment_name
```
Comment (abhiyerra) [inline eks/bastion.tf:52]: Make the two match:

Add the quotes around Name. `"Name"`

File: eks/public_subnet.tf [hunk 2]
```diff
@@ -8,14 +8,16 @@ resource "aws_subnet" "public" {
...
   }
 }
 
 resource "aws_internet_gateway" "public" {
   vpc_id = aws_vpc.vpc.id
 
   tags = {
-    Name = var.environment_name
+    Name                  = var.environment_name
+    "KubespotEnvironment" = var.environment_name
```
Comment (abhiyerra) [inline eks/public_subnet.tf:20]: Same as above comment.

File: eks/public_subnet.tf [hunk 3]
```diff
@@ -28,7 +30,8 @@ resource "aws_route_table" "public" {
     gateway_id = aws_internet_gateway.public.id
   }
   tags = {
-    Name = "${var.environment_name}-public-${count.index}"
+    Name                  = "${var.environment_name}-public-${count.index}"
```
Comment (abhiyerra) [inline eks/public_subnet.tf:33]: Same as above comment.

**Review by abhiyerra — CHANGES_REQUESTED** [review]

File: eks/public_subnet.tf (hunk 3 above)
Comment (janavenkat) [reply eks/public_subnet.tf:33]: Thanks fixed

**Review by abhiyerra — APPROVED** [review]

**Pull request merged**